index.html

```diff
@@ -1028,7 +1028,7 @@ <h2>Algorithm</h2>
 		</ol>
 
 		<section id="dereferencing-algorithm-primary">
-			<h2>Dereferencing the Primary Resource</h2>
+			<h2>Dereferencing the Primary Resource (DID document or other resource)</h2>
```

Comment:
```suggestion
			<h2>Dereferencing the Primary Resource (DID Document or other resource)</h2>
```

do we keep these both caps?